frontend/packages/kubevirt-plugin/src/selectors/vm-template/combined-dependent.ts

```diff
@@ -34,48 +34,46 @@ export const getFlavorLabel = (flavor: string) => {
   return undefined;
 };
 
-export const getOperatingSystems = (templates: TemplateKind[], userTemplate: string) => {
-  if (userTemplate) {
+export const getOperatingSystems = (templates: TemplateKind[], userTemplateName: string) => {
+  if (userTemplateName) {
     return getTemplateOperatingSystems([
-      getTemplatesOfLabelType(templates, TEMPLATE_TYPE_VM).find((t) => getName(t) === userTemplate),
+      getTemplatesOfLabelType(templates, TEMPLATE_TYPE_VM).find(
+        (t) => getName(t) === userTemplateName,
+      ),
     ]);
   }
   return getTemplateOperatingSystems(getTemplatesOfLabelType(templates, TEMPLATE_TYPE_BASE));
 };
 
 export const getWorkloadProfiles = (
   templates: TemplateKind[],
-  { flavor, os, userTemplate }: { flavor: string; os: string; userTemplate: string },
+  { flavor, os, isUserTemplateValid }: { flavor: string; os: string; isUserTemplateValid: boolean },
```

Comment:
why do we need isUserTemplateValid variable? can we just pass templates and that is all?